bgpd/bgp_vty.c

```diff
@@ -8833,6 +8846,86 @@ DEFPY (af_sid_vpn_export,
 	return CMD_SUCCESS;
 }
 
+DEFPY (bgp_sid_vpn_export,
+       bgp_sid_vpn_export_cmd,
+       "[no] sid vpn per-vrf export <(1-255)$sid_idx|auto$sid_auto>",
+       NO_STR
+       "sid value for VRF\n"
+       "Between current vrf and vpn\n"
+       "sid per-VRF (both IPv4 and IPv6 address families)\n"
+       "For routes leaked from current vrf to vpn\n"
+       "Sid allocation index\n"
+       "Automatically assign a label\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	int debug;
+
+	debug = (BGP_DEBUG(vpn, VPN_LEAK_TO_VRF) |
+		 BGP_DEBUG(vpn, VPN_LEAK_FROM_VRF));
+
+	if (no) {
+		/* implement me */
```

Comment:
We need the ability to handle the no form of this command

Reply:
Done. I updated the PR with the implementation of the no form of the command (commit https://github.com/FRRouting/frr/pull/11673/commits/a5e6a474bb39d8b2201c28b9d9c1d16e483d7079). Also, I extended the topotest to test it (commit https://github.com/FRRouting/frr/pull/11673/commits/81e59c62750343a19e75838fef3913e21056a1b9).